### DISCUSSION TITLE
What exactly does project linking do?

### DISCUSSION THREAD

**Discussion by ninest** [vercel/vercel#4490] [Help] [answered]
Project linking docs: https://vercel.com/docs/cli#commands/overview/project-linking

What exactly does this mean? Let's say I have two projects, a Python/Flask API and a Nuxt website. If I "link" the projects, does that mean I can host them under the same `appname.now.sh` domains? Something like `appname.now.sh/` and `appname.now.sh/api`?

**Comment by styfle** [comment, marked as answer]
> What exactly does this mean? 

Project Linking is simply the link between your local project on your file system and the Vercel Project in the cloud.

>  If I "link" the projects, does that mean I can host them under the same appname.now.sh domains? Something like appname.now.sh/ and appname.now.sh/api?

No, this is not possible at this time.

**Reply by ninest** [reply]
Thanks for the answer!
